_data/versions.yml

```diff
@@ -26,7 +26,7 @@ current:
   mirrors: 'https://www.apache.org/dyn/closer.lua/arrow/arrow-9.0.0/'
   tarball-name: 'apache-arrow-9.0.0.tar.gz'
   tarball-url: 'https://www.apache.org/dyn/closer.lua?action=download&filename=arrow/arrow-9.0.0/apache-arrow-9.0.0.tar.gz'
-  java-artifacts: 'http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.apache.arrow%22%20AND%20v%3A%229.0.0%22'
+  java-artifacts: 'https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.apache.arrow%22%20AND%20v%3A%229.0.0%22'
```

Comment:
FYI: This file is generated automatically.
https://github.com/apache/arrow/pull/14329 is for future releases.